source/Projectile.cpp

```diff
@@ -121,6 +126,30 @@ void Projectile::Move(vector<Visual> &visuals, vector<Projectile> &projectiles)
 			for(const auto &it : weapon->Submunitions())
 				for(size_t i = 0; i < it.count; ++i)
 					projectiles.emplace_back(*this, it.offset, it.facing + Projectile::Inaccuracy(it.weapon->Inaccuracy()), it.weapon);
+
+			if(!weapon->ShipToShoot().empty())
+			{
+				const Ship *shipToPlace = GameData::Ships().Get(weapon->ShipToShoot());
+				if(shipToPlace->IsValid())
+				{
+					const shared_ptr<Ship> shotShip = make_shared<Ship>(*shipToPlace);
+					shotShip->Recharge();
+					shotShip->SetName(parentShip->Name());
+					shotShip->SetGovernment(parentShip->GetGovernment());
+					if(weapon->ShipToShootPersonality().IsDefined())
+						shotShip->SetPersonality(weapon->ShipToShootPersonality());
+					else
+						shotShip->SetPersonality(parentShip->GetPersonality());
+					shotShip->SetHail(*parentShip->GetHailPhrase());
+					shotShip->SetSystem(parentShip->GetSystem());
+					if(parentShip->GetParent() && !shotShip->GetPersonality().IsCoward())
```

Comment:
The coward thing only made sense for core ships because they're evacuating, I think. That said, it could be handy to be able to define this fighter as independant so keep it I guess. But mb add a comment?

Reply:
wait, I thought coward ships never have a parent?

Reply:
They only lose the parent when they get low. That's why it made sense for them to do so once evacuating, for the other PR